export.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"strings"
+
+	"notes-service/exports"
+	notespb "notes-service/protorepo/noted/notes/v1"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (srv *notesService) ExportNote(ctx context.Context, in *notespb.ExportNoteRequest) (*notespb.ExportNoteResponse, error) {
+	m := map[string]func(*notespb.Note) ([]byte, error){
+		"markdown": exports.NoteToMarkdown,
+		"pdf":      exports.NoteToPDF,
+	}
+
+	note, err := srv.GetNote(ctx, &notespb.GetNoteRequest{Id: in.NoteId})
+
+	if err != nil {
+		srv.logger.Error("failed to fetch note", zap.Error(err))
```

Comment:
I think it's already logged at the repository level? Am I right in assuming that?